src/transformers/tokenization_utils_base.py

```diff
@@ -1633,6 +1443,9 @@ def __init__(self, **kwargs):
 
         super().__init__(**kwargs)
 
+        self.extra_special_tokens = kwargs.pop("extra_special_tokens", {})
+        self._set_model_specific_special_tokens(special_tokens=self.extra_special_tokens)
```

Comment:
when we do this, we don't add them to the tokenizer vocab  right?

Reply:
I think you are already checking that these tokens are added to the vocab if not already present right?

Reply:
NO it's alright IMO we have not really seen reports about that